### ISSUE TITLE
An aurelia modal without bootstrap

### ISSUE THREAD

**Issue by bastienJS** [PWKad/aurelia-bs-modal#1]
Helloo Patrick,

any chance that we might get a modal without bootstrap/jquery?

I would really appreciate - and I also saw some other requests in aurelia gitter - to have a modal without any UI framework dependency.

:-)

**Comment by plwalters** [comment]
Hey excellent feedback.  I'm curious which would be more desired, a barebones modal or a few different samples using different CSS frameworks.  Any additional feedback welcome.

**Comment by bastienJS** [comment]
That is actually an easy answer. We developers like samples to copy/paste and see it in action. But if its too much work for YOU, just make it barebone working and everyone has to struggle individually with the one or other minor problem ;-)

**Comment by bastienJS** [comment]
Hey Patrick, as I know from gitter aurelia channel that you work on another modal sample maybe you wanna try => http://flexboxgrid.com/

**Comment by plwalters** [comment]
I don't think that flexboxgrid has a modal implementation but I am working on converting the bs modal to be not dependent on Bootstrap

**Comment by delebash** [comment]
I'll put in a vote for a modal example using Materialize, http://materializecss.com/.  I am finding it difficult in getting the plugin to initialize itself as it relies on document.ready to call code like Waves.displayEffect().  Since aurelia loads partial pages document.ready is only called once and not on each class load unless you add an attach handler or inherit a base class that has an attach handler that initializes the code on each page/class load, even with that you have to dig through materialize source to find each initialization code like Waves.displayEffect() for each component, instead of relying on a document.ready trigger, so I would love to see how you approach solving this. 

 If you have time, thanks and thanks for your examples they have been really helpful to me :)

**Comment by bastienJS** [comment]
@PWKad Yes Flexboxgrid is only a grid. A modal is actually only a absolute positioned div with z-index: 1000 so I have not thought that you expect an extra plugin for such a feature. I thought you would implement that manually in the sample :-)

But maybe you should talk to aurelia interface guy and integrate this stuff :-)

**Comment by plwalters** [comment]
The modal you are referencing (such as the z-index being 1000) is not a default modal it is just how bootstraps' modal works.  I was hoping to get some examples of other frameworks that you all would like to see the modal in.  I can put together a pure css version but if anyone has any recommendations on other CSS frameworks please let me know.

**Comment by grantcarthew** [comment]
Hey Patrick, I would like to add my vote to a simple modal without any dependencies from other frameworks.

**Comment by egeland** [comment]
I was told in the Aurelia gitter that https://github.com/aurelia/dialog is the official dialog / modal plugin now, is that correct?
That one appears to be independent of twbs or other frameworks..

If that one _is_ the official one, maybe this one should be marked deprecated and some hint put in the readme to point to `dialog`
